### PR TITLE
ADR for single central schema in the spec and extended schema as issues

### PR DESCRIPTION
**Pull request by tcezard**

### CODE REVIEW

File: docs/decision_record.md [hunk 1]
```diff
@@ -8,6 +8,31 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 [TOC]
 
+## 2024-02-21 Schema definition for the sequence collection attributes and process for adding new attribute
```
Comment (sveinugu) [inline docs/decision_record.md:11]:
```suggestion
## 2024-02-21 Schema definition for the sequence collection attributes and process for adding new attributes
```

File: docs/decision_record.md [hunk 2]
```diff
@@ -8,6 +8,31 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
+
+### Decision
+
+The sequence collection specification will sanction a number of attributes for which a clear and common accepted definition can be found
```
Comment (sveinugu) [inline docs/decision_record.md:15]:
```suggestion
The sequence collection specification will sanction a number of attributes for which a clear and commonly accepted definition will be provided.
```

File: docs/decision_record.md [hunk 3]
```diff
@@ -8,6 +8,31 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
 [TOC]
 
+## 2024-02-21 Schema definition for the sequence collection attributes and process for adding new attribute
+
+### Decision
+
+The sequence collection specification will sanction a number of attributes for which a clear and common accepted definition can be found 
+These attributes will be defined in the sequence collection schema and will be part of the specification.
+
+Additional attributes can be requested to be added to the schema via opening an issue on the sequence collection specification GitHub. These will be tagged with "schema-term"
```
Comment (sveinugu) [inline docs/decision_record.md:18]:
```suggestion
Additional attributes can be requested to be added to the schema via opening an issue on the sequence collection specification GitHub repo. These will be tagged with "schema-term"
```

File: docs/decision_record.md [hunk 4]
```diff
@@ -8,6 +8,31 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
+
+The set of opened tagged issues will provide an extended seqcol schema with all attributes proposed by the community that implementer can monitor to increase forward interoperability
```
Comment (sveinugu) [inline docs/decision_record.md:20]:
```suggestion
The set of open issues with this tag can be viewed as an extended seqcol schema that includes all attributes proposed by the community. It is recommended that implementers monitor this set to increase forward interoperability.
```

File: docs/decision_record.md [hunk 5]
```diff
@@ -8,6 +8,31 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
+
+### Rationale
+
+It is important for the interoperability of services that attributes used in different implementations have the same definition.
+To ensure this, the centrally defined schema will provide clear definition of the most important attributes.
```
Comment (sveinugu) [inline docs/decision_record.md:25]:
```suggestion
To ensure this, the centrally defined schema will provide clear definitions of the most important attributes. 
```

File: docs/decision_record.md [hunk 6]
```diff
@@ -8,6 +8,31 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
+
+The set of opened tagged issues will provide an extended seqcol schema with all attributes proposed by the community that implementer can monitor to increase forward interoperability
+
+### Rationale
+
+It is important for the interoperability of services that attributes used in different implementations have the same definition.
+To ensure this, the centrally defined schema will provide clear definition of the most important attributes. 
+However it is clear that the maintainers cannot define all possible attributes that implementations might need, so it became apparent that an extended list of attributes that have not been fully defined yet would be useful.
+
+Choosing to host this list as a list of issue allows the list to always be up to date and also contain thread where the community can discuss the definition and approval of the attribute.
```
Comment (sveinugu) [inline docs/decision_record.md:28]:
```suggestion
Choosing to host this list as a list of issues allows the list to always be up to date and also contain comment threads where the community can discuss the definition and approval of each attribute.
```

**Review by sveinugu — APPROVED** [review]
Suggested some small language changes. Otherwise, looks good.

File: docs/decision_record.md [hunk 7]
```diff
@@ -8,6 +8,31 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
 [TOC]
 
+## 2024-02-21 Schema definition for the sequence collection attributes and process for adding new attributes
+
+### Decision
+
+The sequence collection specification will sanction a number of attributes for which a clear and commonly accepted definition will be provided.
+These attributes will be defined in the sequence collection schema and will be part of the specification.
+
+Additional attributes can be requested to be added to the schema via opening an issue on the sequence collection specification GitHub repo. These will be tagged with "schema-term"
```
Comment (nsheff) [inline docs/decision_record.md:18]:
```suggestion
Additional attributes can be requested to be added to the schema via opening an issue on the sequence collection specification GitHub repo. These will be labeled "schema-term".
```
```suggestion
Additional attributes can be requested to be added to the schema via opening an issue on the sequence collection specification GitHub repo. These will be tagged with "schema-term"
```

File: docs/decision_record.md [hunk 8]
```diff
@@ -8,6 +8,31 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
+
+The set of open issues with this tag can be viewed as an extended seqcol schema that includes all attributes proposed by the community. It is recommended that implementers monitor this set to increase forward interoperability.
```
Comment (nsheff) [inline docs/decision_record.md:20]:
```suggestion
The set of open issues with this tag can be viewed as an extended seqcol schema that includes all attributes proposed by the community. We RECOMMEND implementers monitor this set to increase forward interoperability.
```

**Review by nsheff — APPROVED** [review]

**Pull request merged**